src/pypromice/L0toL1.py

```diff
@@ -137,22 +139,31 @@ def _flagNAN(ds, flag_file=None):
     ds : xr.Dataset
         Level 0 data with flagged data
     '''
-    # flag_file = "./data/flags/" + ds.attrs["station_id"] + ".csv"
-    if flag_file is None:
-        print('Flag file not given. No data flagged')
+    ds = ds_in.copy()
+    try:
+        os.mkdir('local')
+        os.mkdir('local/flags')
+        os.mkdir('local/adjustments')
+    except:
+        pass
+    flag_url = "https://raw.githubusercontent.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/master/flags/" 
+    try:
+        urllib.request.urlretrieve(flag_url + ds.attrs["station_id"] + ".csv",
```

Comment:
Wondering why this specific approach was chosen. The advantages here are that the flag file is "automatically" read from an external location and written locally so it can be read into pandas (the user doesn't have to think about it).

However, will there ever be any access problems with making a request to raw.githubusercontent.com?

An alternative might be to require cloning of `PROMICE_AWS-data-issues` alongside `pypromice` in the processing directory (in the same way that we clone `aws-l0` and `aws-l3`). Then the flag file can be read from a local directory path.

If we do want to retain the `urllib.request` approah, perhaps the URL should be moved to `args` so it can be easily accessible as a parameter that could possibly change?

This could be totally fine as implemented, just raising some thoughts. The same applies to the similar code in `_adjustData`.

Reply:
Also, after reading the convo on #18, we could also consider using `pd.read_csv()` to read directly from the URL. However, I agree that it will probably be nice to keep an "offline" option for the processing. For example, if the network is down, and you are working on some development, if you have `aws-l0`, `aws-l3` and `pypromice` cloned, you can just run the processing locally. So using either the `local` directory, or requiring `PROMICE_AWS-data-issues` to be cloned is probably the way to go.

Reply:
I think I like the current implementation. Try to fetch from the URL, but if not accessible, then fall back to whatever is in the `local` directory. Makes setup that much easier (no need to clone another repo just for a couple .csv files).

Sorry if I cluttered this with all my thinking! Feel free to resolve if you all agree.

Reply:
I like the idea of trying to fetch from URL and falling back to an offline copy if it's not accessible.

Reply:
This approach has been implemented into the latest commit. 
1. Try to fetch from the URL (assumed to be the most up-to-date)
2. Try to fetch from a local file copy (if URL failed)
3. Flag/adjust if file is successfully loaded as `pd.DataFrame`